vnext/template/cs-app/src/AutolinkedNativeModules.g.props

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
I did some factoring to share as much between cpp and cs as possible.
Please consider if i ti makes sense to place this in `template\shared-app\src` since the file seems identical between cs and cpp

Reply:
Done.